docs/overview.yaml

```diff
@@ -46,6 +49,9 @@ General algebra:
     cyclic group: 'IsCyclic'
     nilpotent group: 'Group.IsNilpotent'
     permutation group of a type: 'Equiv.Perm'
+    alternating group: 'alternatingGroup'
+    A5 is simple: 'alternatingGroup.isSimpleGroup_five'
+    classification of abelian simple groups: CommGroup.is_simple_iff_isCyclic_and_prime_card
```

Comment:
This PR looks good to me, although I'm realizing that this result is a bit strange since prime card implies cyclic...

Reply:
#31829